server/src/main/java/com/battleships/Server.java

```diff
@@ -0,0 +1,93 @@
+package com.battleships;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+class Server {
+    private final ServerSocket serverSocket;
+    private final ConnectedPlayers connectedPlayers;
+    private static final Logger logger = LogManager.getLogger(Server.class.getName());
+
+    Server() throws IOException {
+        logger.info("Próba odpalenia serwera");
+        this.serverSocket = new ServerSocket(50000);
+        logger.info("Serwer odpalony... :)");
+        connectedPlayers = new ConnectedPlayers();
+        tryAcceptPlayersLoop();
+    }
+
+    private void tryAcceptPlayersLoop() {
+        new Thread(() -> {
+            while (true) {
```

Comment:
Is main server loop responsible only for accepting of players?

Reply:
Fixed